Source/Core/DiscIO/VolumeWiiCrypted.cpp

```diff
@@ -237,6 +237,14 @@ bool CVolumeWiiCrypted::IsWiiDisc() const
 	return true;
 }
 
+bool CVolumeWiiCrypted::IsDiscTwo() const
+{
+	bool discTwo = false;
```

Comment:
Changing that might not be as simple as you think. IsDiscTwo is a virtual function in volume.h
I copied this code directly from VolumeGC, save from changing Read() to m_pReader->Read()